labs/PerforamnceSandBox.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: PerformanceSandBox
+parent: Labs
+---
+# Lab Name
+PerformanceSandBox
+
+# Short Description
+The Performance Sandbox Project is a Sandbox project for Hyperledger Projects Performance research usage. It allows user easy to due with performance related works with this sandbox project.
+
+# Scope of Lab
+_Enter the scope of the lab. This should include enough information to allow the Lab stewards to understand how it fits within Hyperledger's mission._
+
+![What is Performance Sandbox](./images/PerformanceSandBox.png "What is Perfomance Sandbox")
+
+## Overall
+As the picture above, it uses kubernetes(as kind) as infrastructure, integrated with monitoring and open tracing(or say open telemetry).
+- It supports user deploy a target network(as test-network for asset transform for Fabric), or called as SUT(system under test).
+- It supports any traffic generator as Tape for Hyperledger Fabric, keep sending traffic to the traget network/SUT.
+Performance researcher able to use this project to do research with steps below:
+1. Start sanbox with k8s integrated with monitoring and open telemetry
+1. Deploy the SUT onto k8s infrastructure.
+1. Start traffic generator to keep sending traffic to SUT.
+1. Monitoring and Analyzing test result from Dash board/open telemetry result.
+1. Changes local source code(for ex Fabric), make local docker image and redeploy the SUT.
+
+It will extend existing PSWG published parper with monitoring and open telemetry. As the right side in the above figure, we can see the perforamnce test model puslished in perviously by PSWG, with SUT, traffic generator and observers.
```

Comment:
```suggestion
It will extend existing PSWG published paper with monitoring and open telemetry. As the right side in the above figure, we can see the performance test model published in previously by PSWG, with SUT, traffic generator and observers.
```